### ISSUE TITLE
Interceptor interfering with home route using Fastify

### ISSUE THREAD

**Issue by jmcdo29** [nestjs/nest#1738]
## I'm submitting a...

<pre><code>
[X] Regression 
[ ] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

When navigating to the home route using the fastify adapter and an interceptor, the home route is not executed. Other routes work fine with the interceptor in place, but not the home route for whatever reason.

## Expected behavior

All routes are executed as expected, even when using interceptors and fastify.

## Minimal reproduction of the problem with instructions

https://github.com/jmcdo29/fasftify-interceptor-issue.git

```sh
git clone https://github.com/jmcdo29/fasftify-interceptor-issue.git
cd fastify-interceptor-issue
npm install
npm run start:dev

# in another terminal
curl http://localhost:3000 
# wait for no response
curl http://localhost:3000/hello
hello world
```

## Side Note
It looks like it may have to do something with res.send() being interfered with by the interceptor. The only issue is I need res.sendFile to work so I can send an html file from the home route. 

## Environment

<pre><code>
Nest version: 6.0.1
 
For Tooling issues:
- Node version: 10.15.3
- Platform:  Linux

</code></pre>

**Comment by jmcdo29** [comment]
After playing around with a few options, it seems that it may be Fastify's reply that is having the issue. If I return an observable or async/await from the route, there are no issues, but using `res.send()` while using the fastify adapter seems to be the problem. Hope this helps track down what's going on.

**Comment by kamilmysliwiec** [comment]
It should be fixed in 6.0.2 :) Thanks for reporting

**Comment by jmcdo29** [comment]
Thanks for the quick turn around! Working as expected!
